src/main/java/tc/oc/pgm/rotation/PGMMapOrderProvider.java

```diff
@@ -0,0 +1,13 @@
+package tc.oc.pgm.rotation;
+
+import tc.oc.pgm.map.PGMMap;
+
+public interface PGMMapOrderProvider {
```

Comment:
Please provide at least some basic documentation.

Reply:
I have added some basic documentation to all of the now refactored classes. Tell me if more is required. https://github.com/Electroid/PGM/pull/114/commits/fe83e3a18ec9419e8184475a0ccd8c4fb5a5e93a